### PR TITLE
Add DE1 SoC support and enabling proginfo for orangecrab

### PR DESCRIPTION
**Pull request by cjearls**

### CODE REVIEW

**Comment by cjearls** [comment]
I mainly just use Git to push, pull, and make new branches, so I wasn't sure how to submit two unrelated pull requests. Instead, they're in a single pull request in two separate commits.

File: blinky/chiselblinky.core [hunk 1]
```diff
@@ -74,6 +79,17 @@ generate:
           - generated/Toplevel.v: { file_type: verilogSource }
           - generated/pll_orange-crab.v: { file_type: verilogSource }
 
+  de1_soc_revF:
+    generator: chisel
+    parameters:
+      extraargs: "-board de1_soc_revF -invreset true"
```
Comment (carlosedp) [inline blinky/chiselblinky.core]: You don't need to pass `-invreset true` since it's the default behavior for the module.

Comment (carlosedp) [reply blinky/chiselblinky.core]: Same could be applied for Orangecrab.

**Review by carlosedp — CHANGES_REQUESTED** [review]

File: blinky/chiselblinky.core (hunk 1 above)
Comment (cjearls) [reply blinky/chiselblinky.core]: Alright, I've removed it from both

**Comment by cjearls** [comment]
Let me know if there's anything else I can improve

**Comment by carlosedp** [comment]
That's great, thanks!

Just a tip, if you prefer to " join" some changes to the previous commit, you could amend it.
You change what you need and use `git commit -v --no-edit --amend`. It would merge these changes into previous commit. the `--no-edit` will reuse previous message.
No need to do here.. just a workflow tip.

**Pull request merged**